dydx/dydxPresenters/dydxPresenters/_v4/Vault/DepositsAndWithdrawals/dydxVaultDepositWithdrawConfirmationViewBuilder.swift

```diff
@@ -29,6 +31,7 @@ private class dydxVaultDepositWithdrawConfirmationViewController: HostingViewCon
 
     override public func arrive(to request: RoutingRequest?, animated: Bool) -> Bool {
         let presenter = presenter as? dydxVaultDepositWithdrawConfirmationViewPresenterProtocol
+        presenter?.amount = request?.params?["amount"] as? Double
```

Comment:
@ruixhuang just pointing out that since abacus does not manage the amount input, we will have to pass inputs around like this during navigation. For more complex inputs, it would be a serializable class. In any case, the lack of typing is a bit of a nuisance for readability.